### PR TITLE
Problem: node startup is slow

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- try iavl lazy mode

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: integration_tests/configs/cosmovisor.jsonnet [hunk 1]
```diff
@@ -5,6 +5,9 @@ config {
     'app-config'+: {
       'minimum-gas-prices': '100000000000basetcro',
     },
+    'app-config'+: {
+      'iavl-lazy-loading':: super.'iavl-lazy-loading',
+    },
```
Comment (mmsqe) [inline integration_tests/configs/cosmovisor.jsonnet:10]:
```suggestion
    'app-config'+: {
      'minimum-gas-prices': '100000000000basetcro',
      'iavl-lazy-loading':: super['iavl-lazy-loading'],
    },
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,7 @@
 ### Improvements
 
 - [#813](https://github.com/crypto-org-chain/cronos/pull/813) Tune up rocksdb options.
+- [#]() Add config iavl-lazy-loading to enable lazy loading of iavl store.
```
Comment (yihuang) [inline CHANGELOG.md:23]:
```suggestion
- [#779](https://github.com/crypto-org-chain/cronos/pull/779) Add config iavl-lazy-loading to enable lazy loading of iavl store.
```

File: integration_tests/configs/cosmovisor.jsonnet [hunk 3]
```diff
@@ -6,6 +6,7 @@ config {
       'minimum-gas-prices': '100000000000basetcro',
       store:: super.store,
       streamers:: super.streamers,
+      'iavl-lazy-loading':: super.'iavl-lazy-loading',
```
Comment (mmsqe) [inline integration_tests/configs/cosmovisor.jsonnet:9]:
```suggestion
      'iavl-lazy-loading':: super['iavl-lazy-loading']
```

**Review by mmsqe — APPROVED** [review]

File: go.mod [hunk 4]
```diff
@@ -190,7 +190,8 @@ require (
 replace (
 	// Ref: https://forum.cosmos.network/t/ibc-security-advisory-dragonberry/7702
 	github.com/confio/ics23/go => github.com/confio/ics23/go v0.9.0
-	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.46.9-0.20230130122755-5e7dcbcce1e9
+  // https://github.com/cosmos/cosmos-sdk/pull/14862
+	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.46.9-0.20230201144439-3b8db0062a59
```
Comment (yihuang) [inline go.mod:194]: wait for the PR a little bit.

**Pull request merged**